.azure/pipelines/quarantined-tests.yml

```diff
@@ -1,13 +1,10 @@
-# We want to run quarantined tests on master as well as on PRs
-trigger:
-  batch: true
-  branches:
-    include:
-    - master
+# We only want to run quarantined tests on master
+pr: none
+trigger: none
 
 schedules:
-- cron: "0 */4 * * *"
-  displayName: Every 4 hours test run
+- cron: "0 */12 * * *"
+  displayName: Every 12 hours test run
```

Comment:
I don't believe we decided to use the same schedule as the Helix-matrix pipeline. Think it was _once_ a day, at a time offset from when that pipeline runs.

Reply:
Ok so once a day at 11 PM work?

Reply:
This works for me but we need more internal agreement (see our chat session)